### PR TITLE
VRF soak test

### PR DESCRIPTION
**Pull request by skudasov**
VRF soak test:
```
make test_vrf_soak
```

### CODE REVIEW

**Review by gheorghestrimtu — APPROVED** [review]

File: suite/performance/vrf.go [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+	}
+}
+
+// RecordValues records VRF metrics
+func (f *VRFTest) RecordValues(b ginkgo.Benchmarker) error {
+	// TODO: collect metrics
+	return nil
+}
+
+// Setup setups OCR performance/soak test
```
Comment (kalverra) [inline suite/performance/vrf.go:89]: `prepares a VRF performance/soak test`

File: suite/performance/vrf.go [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+		}
+	}
+}
+
+// createChainlinkJobs create and collect VRF jobs for every Chainlink node
+func (f *VRFTest) createChainlinkJobs() error {
+	jobsChan := make(chan VRFJobMap, len(f.chainlinkClients)*len(f.contractInstances))
+	g := errgroup.Group{}
+	for _, p := range f.contractInstances {
+		p := p
```
Comment (kalverra) [inline suite/performance/vrf.go:225]: ?

File: suite/performance/vrf.go [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+		for _, n := range f.chainlinkClients {
+			n := n
```
Comment (kalverra) [inline suite/performance/vrf.go:227]: ???

File: suite/performance/vrf_test.go [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+	"github.com/smartcontractkit/integrations-framework/client"
+	"github.com/smartcontractkit/integrations-framework/environment"
+	"github.com/smartcontractkit/integrations-framework/tools"
+	"math/big"
+	"time"
+)
+
+var _ = Describe("VRF soak test @soak-vrf", func() {
+	var (
+		s        *actions.DefaultSuiteSetup
```
Comment (kalverra) [inline suite/performance/vrf_test.go:16]: change `s` to something like `suiteSetup`, makes it more explicit what it actually is referring to

**Review by kalverra — CHANGES_REQUESTED** [review]

**Review by kalverra — APPROVED** [review]

**Review by tateexon — APPROVED** [review]

**Pull request merged**